core/domain/exp_services_test.py

```diff
@@ -2083,6 +2104,21 @@ def test_update_content_missing_key(self):
                     }),
                 '')
 
+    def test_update_written_translations(self):
```

Comment:
I think this earlier comment was missed? 
https://github.com/oppia/oppia/pull/6175#discussion_r253295972